src/commands/pull.js

```diff
@@ -0,0 +1,69 @@
+const chalk = require('chalk');
+const mkdirp = require('mkdirp');
+const fs = require('fs');
+const path = require('path');
+const tiged = require('tiged');
+const { ApifyCommand } = require('../lib/apify_command');
+const { getLoggedClientOrThrow } = require('../lib/utils');
+
+class PullCommand extends ApifyCommand {
+    async run() {
+        const { args } = this.parse(PullCommand);
+        const apifyClient = await getLoggedClientOrThrow();
+        const cwd = process.cwd();
+
+        const { actorId } = args;
+        const actor = await apifyClient.actor(actorId).get();
+        if (!actor) throw new Error(`Cannot find actor with ID '${actorId}' in your account.`);
+        const { name, versions } = actor;
+
+        // TODO: Check is sorted chronologically
+        const lastVersion = versions[versions.length - 1];
+
+        const dirpath = path.join(cwd, name);
+        mkdirp.sync(dirpath);
+
+        switch (lastVersion.sourceType) {
+            case 'TARBALL':
+            case 'SOURCE_FILES': {
+                const { sourceFiles } = lastVersion;
+                for (const file of sourceFiles) {
+                    if (file.folder) mkdirp.sync(path.join(cwd, name, file.name));
+                    else fs.writeFileSync(`${dirpath}/${file.name}`, file.content);
+                }
+                console.log(`${chalk.green('Pulled to')} ${dirpath}/`);
+                break;
+            }
+            case 'GIT_REPO': {
+                // e.g. https://github.com/jakubbalada/Datasety.git#master:RejstrikPolitickychStran
+                const { gitRepoUrl } = lastVersion;
+                const [repoUrl, branchDirPart] = gitRepoUrl.split('#');
+
+                // TODO: Handle defaults and edge cases
+                let branch;
+                let dir;
+                if (branchDirPart) [branch, dir] = branchDirPart.split(':');
```

Comment:
This is a pretty important TODO, it doesn't work in the default case when there is no branch or dir specified, it just silently fails, shouldn't it be done straight away?